src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -487,11 +487,11 @@ void WorldSession::HandleMovementOpcodes(WorldPacket & recvData)
                 }
                 else if (!plrMover->HasFlag(PLAYER_FLAGS, PLAYER_FLAGS_IS_OUT_OF_BOUNDS))
                 {
-                    WorldSafeLocsEntry const* grave = sObjectMgr->GetClosestGraveyard(plrMover->GetPositionX(), plrMover->GetPositionY(), plrMover->GetPositionZ(), plrMover->GetMapId(), plrMover->GetTeamId());
-
-                    if ( grave)
+                    if(WorldSafeLocsEntry const* grave = sObjectMgr->GetClosestGraveyard(plrMover->GetPositionX(), plrMover->GetPositionY(), plrMover->GetPositionZ(), plrMover->GetMapId(), plrMover->GetTeamId()))
```

Comment:
missing space after `if`

Reply:
this can be reverted

Reply:
You can revert the part where you put the declaration of the variable inside the if